src/display/summary-display.ts

```diff
@@ -103,4 +109,32 @@ export class SummaryDisplay {
         this.logger.log(pendingReason.pending);
         this.logger.resetIndent();
     }
+
+    private errorsSummary(errors: CustomReporterResult[]): void {
+        this.logger.log("**************************************************");
+        this.logger.log("*                     Errors                     *");
+        this.logger.log("**************************************************");
+        this.logger.newLine();
+        for (let i = 0; i < errors.length; i++) {
+            this.errorSummary(errors[i], i + 1);
+            this.logger.newLine();
+        }
+        this.logger.newLine();
+        this.logger.resetIndent();
+    }
+
+    private errorSummary(error: CustomReporterResult, index: number): void {
+        this.logger.log(`${index}) ${error.fullName}`);
+        if (this.configuration.summary.displayErrorMessages) {
```

Comment:
These errors are not related to failed assertion, I would be in favor of always displayed it for now